nav2_waypoint_follower/src/waypoint_follower.cpp

```diff
@@ -55,6 +57,10 @@ WaypointFollower::on_configure(const rclcpp_lifecycle::State & /*state*/)
   stop_on_failure_ = get_parameter("stop_on_failure").as_bool();
   loop_rate_ = get_parameter("loop_rate").as_int();
   waypoint_task_executor_id_ = get_parameter("waypoint_task_executor_plugin").as_string();
+  gps_waypoint_transform_timeout_ =
```

Comment:
Woah there, I think you can just store this as a `double`. You end up casting back into a double later anyway to use TF